package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "prepare": "husky install",
-    "generate": "yarn workspace @mesa/generators generate"
+    "generate:component:vue": "yarn workspace @mesa/generators generate"
```

Comment:
I believe we could keep the script as `generate`  and update the plop setGenerator name to something related to what will be generated, like `vue-component`or `component:vue`

Reply:
[✔️](https://emojipedia.org/check-mark/)